Lombiq.DataTables/package.json

```diff
@@ -1,15 +1,49 @@
 {
   "private": true,
-  "scripts": {
-    "dotnet-prebuild": "gulp"
+  "// inactive dependencies - currently replaced by static files with custom fixes": {
+    "datatables.net": "1.10.25",
+    "datatables.net-bs4": "3.2.2"
   },
   "dependencies": {
-    "@babel/preset-env": "^7.14.8",
-    "datatables.net": "1.10.25",
-    "datatables.net-bs4": "3.2.2",
     "datatables.net-buttons": "1.7.1",
     "datatables.net-buttons-bs4": "1.7.1",
-    "gulp-babel": "^8.0.0",
     "urijs": "1.19.7"
```

Comment:
Why are these 3 dependencies kept?

Reply:
Because they were there and active before.

Reply:
My mistake, I thought these are unmanaged too.